encoding/openapi/openapi.go

```diff
@@ -83,6 +86,13 @@ type Config struct {
 	ExpandReferences bool
 }
 
+// SchemaConstraint defines a single entry into a schema.
+// An example would be `SchemaConstraint{Key: "minItems", Attribute: 1}`
```

Comment:
This example isn't valid because the constant `1` isn't an `ast.Expr`.

Also, using an `ast.Expr` allows the encoding of arbitrary CUE expressions: it would be good to describe how they are encoded into the resulting OpenAPI schema.

Reply:
I really just want something that can be be converted into a json expression. Do you think its better to use a `map[?]?` or similar (not sure what type exactly), or use `ast.Expr` but document it better?

Reply:
Done